beacon-chain/blockchain/service.go

```diff
@@ -155,6 +182,11 @@ func (c *ChainService) CurrentActiveState() *types.ActiveState {
 	return c.chain.ActiveState()
 }
 
+// Retrieves the last finalized epcoh of the canonical chain.
+func (c *ChainService) GetLastFinalizedEpoch() uint64 {
```

Comment:
why do we need a another abstraction `GetLastFinalizedEpoch`? when someone can just call `CurrentCrystallizedState().LastFinalizedEpoch()`?